### ISSUE TITLE
Feature Request - Pass view options via query string

### ISSUE THREAD

**Issue by dbornstein** [gvalkov/tailon-legacy#33]
This utility is excellent.  I am using it to enable remote users to view the status of their background jobs.   Tailon is configured to view all the running job logfiles in a certain directory (the auto refresh feature was key)

From my application, I would like to launch tailon and specify which logfile to view so the user doesn't have to select it. Tailon already knows about the files, I just don't want the user to have to select it from the pulldown, I want to launch the UI to display that file as the default for this session.
Something like:  myserver:8080?file=12345.log

**Comment by gvalkov** [comment]
Sounds like a reasonable feature request. I'll look into it. 

On a side note, if you need to expose only a single file to your users, then the [wtee](http://tailon.readthedocs.io/en/latest/#wtee) utility could prove useful.

**Comment by gvalkov** [comment]
Sorry for the huge delay. Implemented in dc94d16dd575b7db81d65188ffa0171bd03f15a1 - I'll rollout a minor release soon.

**Comment by gvalkov** [comment]
Available in 1.1.0. You can now set the initial values through the query string like this:

```
http://localhost:8081/?file=logs/nginx/error.log&cmd=grep&script=GET
```
